modules/persistent-storage-csi-google-cloud-file-create-sc.adoc

```diff
@@ -25,11 +25,13 @@ metadata:
   name: filestore-csi
 provisioner: filestore.csi.storage.gke.io
 parameters:
-  network: network-name <1>
+  connect-mode: DIRECT_PEERING <1>
+  network: network-name <2>
 allowVolumeExpansion: true
 volumeBindingMode: WaitForFirstConsumer
 --
-<1> Specify the name of the GCP virtual private cloud (VPC) network where Filestore instances should be created in.
+<1> Optional. For a shared VPC, use the `connect-node` parameter set to `PRIVATE_SERVICE_ACCESS`. The default setting is `DIRECT_PEERING`, which specifies a non-shared VPC.
+<2> Specify the name of the GCP virtual private cloud (VPC) network where Filestore instances should be created in.
```

Comment:
I think we need to add more details in "2. Specify the name of the VPC network where Filestore instances should be created in." paragraph below (not shown in this PR).
When using a share VPC (i.e connect-mode PRIVATE_SERVICE_ACCESS), the network needs to be the full VPC name. Something like   
`network: projects/shared-vpc-name/global/networks/gcp-filestore-network`

@chao007 to review, comment and confirm

Reply:
Yes, agree.

Reply:
BTW, `connect-mode` is not Optional when we use VPC shared network.

Reply:
Seems like it might be good to reword the descritpiton to be more explicit about the `connect-mode` being optional for the "standard" use-case but required for shared VPC.

Also there is a typo on the line 33 (it says `connect-node`).

Reply:
Fixed.